workflow/test_scores.py

```diff
@@ -0,0 +1,44 @@
+import numpy as np
```

Comment:
Can you move those test files to a `workflow/tests` subdir ?

Reply:
Will do

Reply:
Shall I use relative of absolute paths ?

Reply:
ideally absolute for tests, but since this is not an installed package, relative is probably easier ?

Reply:
yeah right

Reply:
had issues with relative imports, switched to absolute